robot_state: B
cube_a_state: A
cube_b_state: A
